core/domain/stats_domain.py

```diff
@@ -1351,3 +1352,115 @@ def validate(self):
             raise utils.ValidationError(
                 'calculation_output is too big to be stored (size: %d): %s' % (
                     sys.getsizeof(output_data), str(output_data)))
+
+
+class LearnerAnswerDetails(object):
+    """Domain object that represents the answer details submitted by the
+    learner.
+    """
+
+    def __init__(
+            self, state_reference, entity_type, interaction_id,
+            learner_answer_info_list, schema_version=(
+                feconf.CURRENT_LEARNER_ANSWERS_DETAILS_SCHEMA_VERSION)):
+        """Constructs a LearnerAnswerDetail domain object.
+
+            Args:
```

Comment:
Deindent this and the following lines by 4. (In general, please follow the existing style.)

Reply:
Done